source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_cfg.py

```diff
@@ -165,6 +159,16 @@ class SimulationCfg:
     physics_prim_path: str = "/physicsScene"
     """The prim path where the USD PhysicsScene is created. Default is "/physicsScene"."""
 
+    device: str = "cuda:0"
+    """The device to run the simulation on. Default is ``"cuda:0"``.
+
+    Valid options are:
+
+    - ``"cpu"``: Use CPU.
+    - ``"cuda"``: Use GPU, where the device ID is inferred from :class:`~omni.isaac.lab.app.AppLauncher`'s config.
+    - ``"cuda:N"`` Use GPU, where N is the device ID.
```

Comment:
```suggestion
    - ``"cpu"``: Use CPU.
    - ``"cuda"``: Use GPU, where the device ID is inferred from :class:`~omni.isaac.lab.app.AppLauncher`'s config.
    - ``"cuda:N"`` Use GPU, where N is the device ID. For example, "cuda:0".
```

Reply:
Would be great to link to the AppLauncher doc on how the device ID is inferred there?